rules/combo/exploit/breakout.yara

```diff
@@ -11,21 +13,23 @@ rule probable_container_breakout : high {
     hash_2024_enumeration_deepce = "76b0bcdf0ea0b62cee1c42537ff00d2100c54e40223bbcb8a4135a71582dfa5d"
     hash_2024_enumeration_linpeas = "210cbe49df69a83462a7451ee46e591c755cfbbef320174dc0ff3f633597b092"
   strings:
-    $docker_sock = "docker.sock"
-    $map_root_to_mnt = "/:/mnt"
-    $docker_mount_root = /docker run[ \w\-]* -v \/:/
-    $cp_bin_bash = "cp /bin/bash"
-    $fdisk_grep = "fdisk -l | grep"
-    $proc1_status = "/proc/1/status"
-    $capsh_print = "capsh --print"
-    $capsysadmin = "CAP_SYS_ADMIN"
-    $notify_on_release = "notify_on_release"
-    $apparmor = "/sys/kernel/security/apparmor"
-    $mount_cgroup = "mount -t cgroup"
-    $rdma_cgroup = "rdma cgroup"
-    $cgroup_procs = "cgroup.procs"
+    $x_docker_sock = "docker.sock"
+    $x_map_root_to_mnt = "/:/mnt"
+    $x_docker_mount_root = /docker run[ \w\-]* -v \/:/
+    $x_cp_bin_bash = "cp /bin/bash"
+    $x_fdisk_grep = "fdisk -l | grep"
+    $x_proc1_status = "/proc/1/status"
+    $x_capsh_print = "capsh --print"
+    $x_capsysadmin = "CAP_SYS_ADMIN"
+    $x_notify_on_release = "notify_on_release"
+    $x_apparmor = "/sys/kernel/security/apparmor"
+    $x_mount_cgroup = "mount -t cgroup"
+    $x_rdma_cgroup = "rdma cgroup"
+    $x_cgroup_procs = "cgroup.procs"
+
+	$not_systemd = "SYSTEMD_PROC_CMDLINE"
   condition:
-    3 of them and not containerd
+    4 of them and not container_managers
```

Comment:
Does this rule need a `none of ($not*)` condition to account for `$not_systemd`?

Reply:
Good catch; done.